tsl/src/continuous_aggs/create.c

```diff
@@ -959,6 +959,26 @@ cagg_agg_validate(Node *node, void *context)
 static bool
 cagg_query_supported(Query *query, StringInfo hint, StringInfo detail, const bool finalized)
 {
+/*
+ * For now deprecate partial aggregates on release builds only.
+ * Once migration tests are made compatible with PG15 enable deprecation
+ * on debug builds as well.
+ */
+#ifndef DEBUG
```

Comment:
Just a comment... use the DEBUG flag is much better than create a GUC... very good idea!!! Perhaps in the future we can invent an specific build flag for deprecating features (aka ENABLE_DEPRECATED_FEATURES) but what you did is enough for now!!!

Reply:
Thanks.